src/core/modules/NomenclatureMgr.cpp

```diff
@@ -558,6 +554,18 @@ int NomenclatureMgr::getTerminatorMaxAltitude() const
 	return NomenclatureItem::terminatorMaxAltitude;
 }
 
+void NomenclatureMgr::setFlagOutlineCraters(bool b)
+{
+	NomenclatureItem::flagOutlineCraters = b;
+	conf->setValue("astro/flag_planets_nomenclature_outline_craters", b);
```

Comment:
This immediate saving is inconsistent with the rest of the options here (except the terminator ones, which are also inconsistent with the rest).

Reply:
Ops... I agree

Reply:
I know. I like the immediate storage of details much better than having to remember whether I have made any other detail changes before I save all options. Immediate here are color, terminator, solar altitudes, and now circles. The global (not-immediate) option is nomenclature or not.  But indeed I would like to have many more immediate-store settings.

Reply:
This inconsistency is very confusing.

Reply:
Immediate storage of details is very bad for educational purposes (esp. for using Stellarium in classes) - it's good for personal uses only. Please move saving option into ConfigurationDialog class.

Or, at least, we should add support profiles - #439